cmd/support-top-locks.go

```diff
@@ -104,7 +102,31 @@ func (u lockMessage) String() string {
 // JSON jsonified top oldest locks message.
 func (u lockMessage) JSON() string {
 	u.Status = "success"
-	statusJSONBytes, e := json.MarshalIndent(u, "", " ")
+	type lockEntry struct {
```

Comment:
If I am not mistaken, you can add duration to lockMessage and calculate it inside mainSupportTopLocks(), just a little simpler

Reply:
tl;dr I believe that elapsed being a human-friendly property of a lock groups the different properties of a lock just like the plain-text output. So, the additional code is worth it.

I agree with you that including the time for which a lock was held in `lockMessage` struct reduces lines of code, but it leaves us with the following json output, which doesn't follow from the plain-text output (shown below).
```
{
 "status": "success",
 "locks": {
  "time": "2022-11-16T19:49:15.86049989Z",
  "elapsed": 13384254295484,
  "resource": ".minio.sys/leader.lock",
  "type": "WRITE",
  "source": "[shared-lock.go:38:sharedLock.backgroundRoutine()]",
  "serverlist": [
   "minio2:9000",
   ...
   "minio1:9000"
  ],
  "owner": "minio1:9000",
  "id": "d4a4b188-507e-497f-8a1f-f8f2ad6af1e8",
  "quorum": 3
 },
 "time": "3 hours"
}
...
``` 
Note: Columns `Time`, `Type` and `Resource` are a lock's properties. With this suggested change, the `time` property would be associated with the `lockMessage`. 
```
Time                  Type    Resource
3 hours               WRITE   .minio.sys/leader.lock
...
```